onnxruntime/contrib_ops/cpu/optional_ops/optional_ops.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+class Optional final : public OpKernel {
```

Comment:
>lass Optional final [](http://example.com/codeflow?start=1&length=19)

Can not find the def file and there is not a documentation what it does.

Reply:
The def file was checked in in an earlier PR by Negin